appveyor.yml

```diff
@@ -1,31 +1,65 @@
-image: Visual Studio 2019
-
 shallow_clone: true
 
 platform:
     - x64
 
 environment:
     matrix:
-        - PYTHON_VERSION: 3.7
+        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
           MINICONDA_DIRNAME: C:\Miniconda37-x64
-        - PYTHON_VERSION: 3.8
+          PYTHON_VERSION: 3.7
```

Comment:
Do we need to test on so many versions? Same tests (minus mypy/black) are execute with azure. Maybe just a single python version would be enough and would definitely decrease the runtime.

Reply:
So, Python 3.8?

Reply:
I guess maybe just let conda pick the version? That would simplify things for me.

Reply:
Sounds good